### PR TITLE
TE-242 FeaturedProperty widget

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-242)

### What **one** thing does this PR do?
Creates component, examples and tests for FeaturedProperty

### CODE REVIEW

File: src/components/widgets/FeaturedProperty/Readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+```jsx
+const { twoCats } = require('./mock-data/images');
```
Comment (davidpelayo) [inline src/components/widgets/FeaturedProperty/Readme.md]: The only downside of this method is the end user doesn't actually see the code behind the scenes

File: src/components/widgets/FeaturedProperty/component.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  imageUrl,
+  guestsNumber,
+  locationName,
+  nightPrice,
+  onClick,
+  propertyName,
+  propertyType,
+  ratingNumber,
+}) => (
+  <Card as="div" onClick={onClick}>
```
Comment (davidpelayo) [inline src/components/widgets/FeaturedProperty/component.js]: A doubt incoming in my mind.
For the matter of logging/tracking services, I think we should dispatch user action events upwards just after processing the event first.
This means capturing this `onClick`, doing something with it (meaning a middle-man handler) and dispatching it to `props.onClick`.
What do you think?

File: src/semantic/src/themes/livingstone/views/card.overrides [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+/* Icons */
+
+/* Links */
+
+/* Description */
+  .description {
+    margin-top: @descriptionDistance;
+
+    h5.ui.header {
```
Comment (davidpelayo) [inline src/semantic/src/themes/livingstone/views/card.overrides]: Why is there the need of `h5.ui.header`?

File: src/components/widgets/FeaturedProperty/component.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    <Image src={imageUrl} />
```
Comment (davidpelayo) [inline src/components/widgets/FeaturedProperty/component.js]: Alternative texts are always good for 🙈 people

File: src/components/widgets/FeaturedProperty/Readme.md (hunk 1 above)
Comment (marcdavi-es) [reply src/components/widgets/FeaturedProperty/Readme.md]: Yes - I think we need to make our approach to mocking more consistent. Probably a separate lump of work?

File: src/semantic/src/themes/livingstone/views/card.overrides (hunk 3 above)
Comment (marcdavi-es) [reply src/semantic/src/themes/livingstone/views/card.overrides]: For the price :)

screen shot 2018-03-27 at 15 27 32

File: src/components/widgets/FeaturedProperty/component.js (hunk 2 above)
Comment (davidpelayo) [reply src/components/widgets/FeaturedProperty/component.js]: I think we can remove it for now. However, have please a look at [this](https://github.com/lodgify/lodgify-ui/pull/36#pullrequestreview-107291187).

File: src/components/widgets/FeaturedProperty/Readme.md (hunk 1 above)
Comment (davidpelayo) [reply src/components/widgets/FeaturedProperty/Readme.md]: Yes, but let's forget it for now.

File: src/components/widgets/FeaturedProperty/component.js (hunk 2 above)
Comment (marcdavi-es) [reply src/components/widgets/FeaturedProperty/component.js]: I'll take it out.

**Pull request merged**